src/config/constants/farmAuctions.ts

```diff
@@ -0,0 +1,165 @@
+import { FarmAuctionBidderConfig } from './types'
```

Comment:
This file stores projects data that is not optimal to store on SC.  It is accessed via `getBidderInfo` and fallbacks to UNKNOWN_BIDDER if it is whitelisted in SC but doesn't have entry here (should not happen in pracite, we must update it when we whitelist wallets in SC)